src/model/rpc.js

```diff
@@ -47,7 +47,7 @@ module.exports = class BtpRpc extends EventEmitter {
     this.debug = debug
   }
 
-  async addSocket (socket, authToken) {
+  async addSocket (socket, authUsername, authToken) {
```

Comment:
if we have 3 positional arguments we should probably change this to take an object. Or it could be `(socket, auth)` where `auth` contains `username` and `token`. That way `isClient` can be `Boolean(auth)`.